### ISSUE TITLE
Keras sparse_categorical_crossentropy is not supported

### ISSUE THREAD

**Issue by bowbahdoe** [eclipse/deeplearning4j#6377]
I am using version `1.0.0-beta2` and when I try to import a simple CNN from keras I get the following error directing me to open an issue. 

```
CompilerException org.deeplearning4j.nn.modelimport.keras.exceptions.UnsupportedKerasConfigurationException: Loss function sparse_categorical_crossentropy not supported yet.. Please file an issue at http://github.com/deeplearning4j/deeplearning4j/issues., compiling:(form-init5429027263786851402.clj:1:12) 
```

Model code:

```python
model = tf.keras.models.Sequential()


model.add(tf.keras.layers.Conv2D(64, (3, 3), input_shape=(200, 200, 1)))
model.add(tf.keras.layers.Activation("relu"))
model.add(tf.keras.layers.MaxPooling2D(pool_size=(2,2)))

model.add(tf.keras.layers.Conv2D(64, (3, 3)))
model.add(tf.keras.layers.Activation("relu"))
model.add(tf.keras.layers.MaxPooling2D(pool_size=(2,2)))

model.add(tf.keras.layers.Flatten())
model.add(tf.keras.layers.Dense(32))

model.add(tf.keras.layers.Dense(len(categories.keys())))
model.add(tf.keras.layers.Activation("softmax"))

model.compile(
    loss="sparse_categorical_crossentropy",
    optimizer="adam",
    metrics=["accuracy"]
)
```

Loading code:

```clojure
(ns com.halobot.classifier
  (:import [org.deeplearning4j.nn.modelimport.keras KerasModelImport]
           [org.nd4j.linalg.io ClassPathResource]))

(defn load-seq-model-with-weights [resource-path]
  (let [model-path (-> (ClassPathResource. resource-path)
                       (.getFile)
                       (.getPath))]
    (KerasModelImport/importKerasSequentialModelAndWeights model-path)))

(def weapon-classifier (load-seq-model-with-weights "weapon_classifier.h5"))
```

**Comment by maxpumperla** [comment]
thanks for spotting this. will get around to this next week.

**Comment by AlexDBlack** [comment]
@maxpumperla FYI this was added a week ago, not sure if it's useful or not here?
https://github.com/deeplearning4j/deeplearning4j/blob/master/libnd4j/include/ops/declarable/generic/loss/sparseSoftmaxCrossEntropyWithLogits.cpp

**Comment by maxpumperla** [comment]
@AlexDBlack thanks, yeah we can pick this up and make it a proper `Loss` that can be imported

**Comment by SandeepKumarJoshi** [comment]
@maxpumperla / @AlexDBlack : I have tried with 1.0.0-beta2 and 1.0.0-beta3 version but this issue is not yet resolved. I am getting same error.

**Comment by maxpumperla** [comment]
@SandeepKumarJoshi didn't get around to it yet.

**Comment by SandeepKumarJoshi** [comment]
@maxpumperla : When can I expect the resolution of this bug?

**Comment by HerrSchwarz** [comment]
Problem still exists using 1.0.0-beta5. Maybe you could fix the documentation here: https://deeplearning4j.org/docs/latest/keras-import-losses

and maybe here too: https://deeplearning4j.org/docs/latest/keras-import-supported-features

**Comment by moberc** [comment]
I have also hit this problem with 1.0.0-beta5.

**Comment by AlexDBlack** [comment]
Thanks for the report, and sorry this took so long. Looks like it just fell through the cracks.

A fix has been merged here: https://github.com/KonduitAI/deeplearning4j/pull/73

It will be merged back to eclipse/deeplearning4j shortly, and will be available on snapshots soon after that.
https://deeplearning4j.org/docs/latest/deeplearning4j-config-snapshots
It will of course also be available in the next release (which we are working towards at present).

Edit: it's available on eclipse/deeplearning4j master now.

**Comment by moberc** [comment]
Many thanks – I’ll test this on Monday!

**Comment by moberc** [comment]
Hi,

I was able to verify the fix against the current snapshot.

Many thanks,
  Margaret
